java/src/main/java/vw/jni/NativeUtils.java

```diff
@@ -88,7 +88,7 @@ else if (osName.toLowerCase().contains("linux")) {
             if (version == null) {
                 throw new UnsupportedOperationException("Cannot determine linux version");
             }
-            return distro.trim().replaceAll(" ", "_") + "." + version;
+            return (distro.trim().replaceAll(" ", "_") + "_" + version).toLowerCase();
```

Comment:
The rationale behind proposing a naming scheme change was that the syntax would allow easier parsing of different name components. Consider the following example based on the original scheme:
`vw_jni.ubuntu.13.10.amd64.lib`
arguably this would be somewhat trickier to parse than 
`vw_jni-ubuntu_13.10_amd64.lib`

So, that was the rationale, but I don't have a problem with using the original scheme either, if you think that it would be better.

Also, `os-maven-plugin` lowercases strings, so I think we should follow that convention.